gapis/api/vulkan/find_issues.go

```diff
@@ -295,7 +295,7 @@ func (t *findIssues) Flush(ctx context.Context, out transform.Writer) error {
 				return
 			}
 			// The debug report is issued for a trace command
-			issue.Command = api.CmdID(label - uint64(t.numInitialCmds))
+			issue.Command = api.CmdID(label)
```

Comment:
I don't think this change is a good idea. `issue.Command` should remain a valid command ID. To me it looks like this change just adds an offset of `t.numInitialCmds` here, just to remove it again when resolving the report.

Reply:
Well, it is actually the other way around if you look at the rest of the code. 

The label here is a cmdID counted with the initial commands. We used to convert it by substracting the number of the initial commands. 

If we do the conversion here, we have no way to understand that if it's an issue coming find_issues.go with a cmdID without initial commands  or if it's an issue coming from externs.go with a cmdID including initial commands. We cannot resolve that at that point.